android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/responses/ShowConstantTbtResponseTests.java

```diff
@@ -21,7 +21,7 @@
 
 /**
  * This is a unit test class for the SmartDeviceLink library project class :
- * {@link com.smartdevicelink.proxy.rpc.ShowConstantTbtResponse}
+ * {@link ShowConstantTbtResponse}
```

Comment:
@kostyaBoss I would delete line 24 and replace it with its old line `{@link com.smartdevicelink.proxy.rpc.ShowConstantTbtResponse}`